sherpa/astro/io/crates_backend.py

```diff
@@ -98,28 +105,36 @@ def get_filename_from_dmsyntax(filename, blockname=None):
             dmsyn = parts.pop(0).lower()
 
     if not is_binary_file(filename):
-        isbinary = False
-        # TODO: set encoding='UTF-8' or maybe 'ascii'
-        with open(filename, mode='r') as fh:
+        colnames = True
+
+        # Crates only guarantees to support ASCII-like variants but it
+        # should be okay to use a less-restrictive encoding here.
+        #
+        with open(filename, mode='r', encoding="UTF8") as fh:
             last = None
             line = fh.readline().strip()
-            while len(line) > 0 and line[0] in '#%':
+            while len(line) > 0 and line.startswith("#"):
                 last = line
```

Comment:
This is change in behavior (`%` is no longer allowed). Do we need to document that?

Reply:
So, this is an interesting one (and I think easier to see in smaller PRs like this than the previous PR). The safest thing is probably just to keep the old behavior, but we have never really discussed '%' support elsewhere and so I thought we could see what happens if we remove it. So we could just document the change. Or try to improve the code.

Reply:
Agreed; as few notes as we can get away with. I've never seen this used, any maybe nobody ever did.